scripts/should-build.rb

```diff
@@ -0,0 +1,108 @@
+require 'json'
+
+PLATFORM = ENV.fetch('task', 'unknown')
```

Comment:
I prefer the following syntax:
```suggestion
PLATFORM = ENV['task'] || 'unknown'
```

Also, freeze it?

```suggestion
PLATFORM = (ENV['task'] || 'unknown').freeze
```

Reply:
Rubocop said something about `Do not freeze immutable objects, as freezing them has no effect.` on the regexp one… I can still freeze this one too

Reply:
Oh, no need to do that.  We should not freeze the regexp literal then.

Reply:
Oh, that went away after upgrading rubocop.

Reply:
Oh, then we should freeze it this one too :P